app/views/datasets/_meta_data.html.erb

```diff
@@ -13,15 +12,15 @@
             <dt>Last updated:</dt>
             <dd><%= format(@dataset["updated_at"]) %></dd>
             <dt>Expected update:</dt>
-              <% if @dataset["datafiles"].any? %>
-                <dd class="<%= expected_update_class_for(@dataset['frequency'])%>">
+            <% if @dataset["datafiles"].any? %>
+                <dd class="<%= expected_update_class_for(@dataset['frequency']) %>">
```

Comment:
I see it's a setting called 'continuation indent' which is currently set to 4 spaces. If I set to 2 will this be consistent with your IDE?